src/components/Badge/tests/Badge.test.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Badge from '../Badge';
+
+describe('<DisplayText />', () => {
+
+    it('Verify the default badge.', () => {
+      const badgeWrapper = mount(<Badge>Status of badge is default</Badge>);
+      expect(badgeWrapper.prop('status')).toBeFalsy();
+      });
+
+    it('Verify the Success badge.', () => {
+      const badgeWrapper = mount(<Badge status="success">Status of badge is success.</Badge>);
+      expect(badgeWrapper.prop('status')).toBe('success');
+      });
+
+    it('Verify the Informational badge.', () => {
+      const badgeWrapper = mount( <Badge status="info">Status of badge is info.</Badge>);
+      expect(badgeWrapper.prop('status')).toBe('info');
+      });
+
+    it('Verify the Attention badge.', () => {
+      const badgeWrapper = mount(<Badge status="attention">Status of badge is attention.</Badge>);
+      expect(badgeWrapper.prop('status')).toBe('attention');
+      });
+
+      it('Verify the Warning badge.', () => {
+      const badgeWrapper = mount(<Badge status="warning">Status of badge is warning.</Badge>);
+      expect(badgeWrapper.prop('status')).toBe('warning');
+      });
+
+    it('Verify the children prop of badge.', () => {
+      const badgeWrapper = mount(<Badge status="warning">Status of badge is warning.</Badge>);
+      expect(badgeWrapper.prop('children')).toBe('Status of badge is warning.');
+      });
+
+    it('Verify the children prop of badge.', () => {
+      const badgeWrapper = mount(<Badge children="Hardik Mehta"></Badge>);
+      expect(badgeWrapper.prop('children')).toBe('Hardik Mehta');
+      });
+
+    it('Verify the children prop of badge is blank.', () => {
+      const badgeWrapper = mount(<Badge status="warning" children=""></Badge>);
+      expect(badgeWrapper.prop('children')).toBeDefined;
```

Comment:
toBeDefined method should be invoked - expect(badgeWrapper.prop('children')).toBeDefined();